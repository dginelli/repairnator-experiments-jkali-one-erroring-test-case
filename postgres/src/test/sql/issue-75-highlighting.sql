SELECT DISTINCT "fieldName"
FROM zdb_highlight('so_posts', '#bool(#must(title:java javascript))',
                   $$ zdb('so_posts', ctid)==>'#bool(#must(java javascript))' $$) ORDER BY 1;