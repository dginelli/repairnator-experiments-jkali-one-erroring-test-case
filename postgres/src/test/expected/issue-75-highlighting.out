SELECT DISTINCT "fieldName"
FROM zdb_highlight('so_posts', '#bool(#must(title:java javascript))',
                   $$ zdb('so_posts', ctid)==>'#bool(#must(java javascript))' $$) ORDER BY 1;
 fieldName 
-----------
 body
 tags
 title
(3 rows)

